common/list/list-group-en.html

```diff
@@ -0,0 +1,167 @@
+</div>
+<h4>Code</h4>
+<pre><code>&lt;p&gt;No links:&lt;/p&gt;
+&lt;ul class="list-group"&gt;
+	&lt;li class="list-group-item list-group-item-success"&gt;Text&lt;/li&gt;
+	&lt;li class="list-group-item list-group-item-info"&gt;Text&lt;/li&gt;
+	&lt;li class="list-group-item list-group-item-warning"&gt;Text&lt;/li&gt;
+	&lt;li class="list-group-item list-group-item-danger"&gt;Text&lt;/li&gt;
+&lt;/ul&gt;
+&lt;p&gt;With links:&lt;/p&gt;
+&lt;div class="list-group"&gt;
+	&lt;a href="#" class="list-group-item list-group-item-success"&gt;Link text&lt;/a&gt;
+	&lt;a href="#" class="list-group-item list-group-item-info"&gt;Link text&lt;/a&gt;
+	&lt;a href="#" class="list-group-item list-group-item-warning"&gt;Link text&lt;/a&gt;
+	&lt;a href="#" class="list-group-item list-group-item-danger"&gt;Link text&lt;/a&gt;
+&lt;/div&gt;
+&lt;div class="list-group"&gt;
+	&lt;a href="#" class="list-group-item list-group-item-success active"&gt;Link text (active item)&lt;/a&gt;
+	&lt;a href="#" class="list-group-item list-group-item-info active"&gt;Link text (active item)&lt;/a&gt;
+	&lt;a href="#" class="list-group-item list-group-item-warning active"&gt;Link text (active item)&lt;/a&gt;
+	&lt;a href="#" class="list-group-item list-group-item-danger active"&gt;Link text (active item)&lt;/a&gt;
+&lt;/div&gt;</code></pre>
+
+<h3>Badges</h3>
+<div class="panel-body">
+	<p>No links:</p>
+	<ul class="list-group">
+		<li class="list-group-item"><span class="badge">14</span> Text</li>
+		<li class="list-group-item"><span class="badge">2</span> Text</li>
+	</ul>
+	<p>With links:</p>
+	<div class="list-group">
+		<a href="#" class="list-group-item"><span class="badge">14</span> Link text</a>
+		<a href="#" class="list-group-item active"><span class="badge">2</span> Link text (active item)</a>
+		<a href="#" class="list-group-item"><span class="badge">5</span> Link text</a>
+	</div>
+</div>
+<h4>Code</h4>
+<pre><code>&lt;div class="panel-body"&gt;
+	&lt;p&gt;No links:&lt;/p&gt;
+	&lt;ul class="list-group"&gt;
+		&lt;li class="list-group-item"&gt;&lt;span class="badge"&gt;14&lt;/span&gt; Text&lt;/li&gt;
+		&lt;li class="list-group-item"&gt;&lt;span class="badge"&gt;2&lt;/span&gt; Text&lt;/li&gt;
+	&lt;/ul&gt;
+	&lt;p&gt;With links:&lt;/p&gt;
+	&lt;div class="list-group"&gt;
+		&lt;a href="#" class="list-group-item"&gt;&lt;span class="badge"&gt;14&lt;/span&gt; Link text&lt;/a&gt;
+		&lt;a href="#" class="list-group-item active"&gt;&lt;span class="badge"&gt;2&lt;/span&gt; Link text (active item)&lt;/a&gt;
+		&lt;a href="#" class="list-group-item"&gt;&lt;span class="badge"&gt;5&lt;/span&gt; Link text&lt;/a&gt;
+	&lt;/div&gt;
+&lt;/div&gt;</code></pre>
+
+<h3>Complex content </h3>
+<p> The following content are not compliant to accessibility and they are there only for example purpose. skip this non-accessible content <a href="#skipComplexContent">(link to the following section)</a></p>
```

Comment:
The text link need to be update at our context.

```suggestion
<p> The following content are not compliant to accessibility and they are there only for example purpose. <a href="#skipComplexContent">Skip this non-accessible content section.</a></p>
```